### ISSUE TITLE
[question] howto customize logger

### ISSUE THREAD

**Issue by chris-aeviator** [turt2live/matrix-js-bot-sdk#81]
When using the SDK it becomes very hard to follow console output, since there is a great level of verbosity about matrix sync enabled by standard when using the matrix client.

I tried redefining mXlogger like suggested in https://github.com/matrix-org/matrix-js-sdk/issues/1059#issuecomment-599123932 but with no effect

**Comment by turt2live** [comment]
There's a `setLogger` function.

Please use #matrix-bot-sdk:t2bot.io for support.

**Comment by chris-aeviator** [comment]
@turt2live thank you for the quick and helpful reply, I will exclusively use the matrix room for future support.

For anybody in the future having the same issue here's how I did it

```
var botSdk = require('matrix-bot-sdk')
// ...

var LogService = botSdk.LogService
// loglevel has type safe settings via e.g. log.levels.SILENT but I could not find them for the moment
LogService.setLevel('silent')

// ....
const matrixClient = new MatrixClient(homeserverUrl, accessToken, storage);

```

**Comment by woifes** [comment]
I also would appreciate the possibility to activate and deactive the logging. Alot of packages use the package [debug](https://www.npmjs.com/package/debug). In this package you can control which messages are logged by environment variables.
